res/css/views/rooms/_TopUnreadMessagesBar.pcss

```diff
@@ -22,60 +22,73 @@ limitations under the License.
     top: 24px;
     right: 24px;
     width: 38px;
-}
 
-.mx_TopUnreadMessagesBar::after {
-    content: "";
-    position: absolute;
-    top: -8px;
-    left: 10.5px;
-    width: 4px;
-    height: 4px;
-    border-radius: 16px;
-    background-color: $secondary-accent-color;
-    border: 6px solid $accent;
-    pointer-events: none;
-}
+    &::after {
+        content: "";
+        position: absolute;
+        top: -8px;
+        left: 10.5px;
+        width: 4px;
+        height: 4px;
+        border-radius: 16px;
+        background-color: $secondary-accent-color;
+        border: 6px solid $accent;
+        pointer-events: none;
+    }
 
-.mx_TopUnreadMessagesBar_scrollUp {
-    height: 38px;
-    border-radius: 19px;
-    box-sizing: border-box;
-    background: $background;
-    border: 1.3px solid $muted-fg-color;
-    cursor: pointer;
-}
+    /* Define common declarations. Their values are specified below. */
+    .mx_TopUnreadMessagesBar_scrollUp,
+    .mx_TopUnreadMessagesBar_markAsRead {
+        background: var(--background);
+        width: var(--width);
+        height: var(--height);
+        border: var(--border);
+        border-radius: var(--borderRadius);
```

Comment:
Please note that using names which seem general for variables does not matter here, as the custom properties cascade. Even if `--height`, for example, is specified somewhere outside of this file, its value is not be respected, because the value for the custom property is explicitly specified below for each selector (`--height: 38px` and `--height: 18px`).